index.bs

```diff
@@ -972,22 +972,22 @@ defined in [[!WebCryptoAPI]].
 
 ## Algorithm to Relax the Same-Origin Restriction ## {#algo-relax-same-origin-restriction}
 
-In general, the same-origin policy prevents web applications from interacting
-within user agents, unless they have the same origin. However, there are many
-specific facets and exceptions to this overall policy.
+In general, the same-origin policy prevents pages from different origins from
+interacting with each other. However, for payments we want to allow subdomains
+to create payment requests that look like they come from the main domain.
```

Comment:
I don't know where the "payments" term came from and do not think we should use that term here.  I would delete that sentence since the sentence below essentially covers that base.